### ISSUE TITLE
Add Support for ARM64V8 architecture 

### ISSUE THREAD

**Issue by odidev** [docker-flink/docker-flink#40]
Hi,

The Flink docker package is building and running fine at my end for ARM64v8 architecture.

Branch used to build is: 
````
1.5/scala_2.11-debian
````
Please suggest, what needs to be done to raise the request to add support for ARM64v8 arch at official docker library too.

Regards,

**Comment by patricklucas** [comment]
This seems reasonable—I'll go ahead and make the change.

@odidev @lag-linaro: when the ARM64v8 images are available, would either/both of you mind spot testing a few of the Flink/Hadoop version combinations? I don't have convenient access to ARM hardware at the moment.

**Comment by patricklucas** [comment]
https://github.com/docker-library/official-images/pull/4569

**Comment by lag-linaro** [comment]
Yes, just provide me with some instructions and I'll happily test.

**Comment by patricklucas** [comment]
Thanks. Running a test job on the four combinations of {1.4,1.5}-{debian,alpine} should be sufficient.

**Comment by iemejia** [comment]
That we are getting multiarch docker images sounds fantastic even worth mentionin in flink's dev@ !
@patricklucas maybe it is worth to add the raspberry pi arch there too, I can test that one.  I was wondering how can we just automatically test the base x86 images with that in place we can then extend it to ARM ones. Any ideas?

**Comment by iemejia** [comment]
Filled #41 for the other arch support

**Comment by patricklucas** [comment]
These images are available now. @odidev / @lag-linaro can you try them out?

**Comment by lag-linaro** [comment]
`docker pull flink`
`docker run -p 8081:8081 -t flink local`

These commands succeed and it is possible to browse to http://<server_ip>:8081

Tested-by: Lee Jones <lee.jones@linaro.org>
